dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/asset/ServerAsset.java

```diff
@@ -292,17 +291,19 @@ public String run() {
         private static EmbeddedServerImpl startServer(File serverXML, String serverName, Map<String, String> serverProps, boolean isTempServer) throws InstallException {
             validateFile(serverXML);
 
-            ServerBuilder sb = new ServerBuilder().setName(serverName);
+            File outputDir = null;
+            File logDir = null;
+            String workareaDirStr = null;
             if (isTempServer) {
-                sb.setOutputDir(serverXML.getParentFile().getParentFile());
+                outputDir = serverXML.getParentFile().getParentFile();
             } else {
-                sb.setOutputDir(Utils.getOutputDir());
+                outputDir = Utils.getOutputDir();
                 InstallLogUtils.getInstallLogger().log(Level.FINEST, "Set output dir to " + Utils.getOutputDir());
-                sb.setLogDir(Utils.getLogDir());
+                logDir = Utils.getLogDir();
                 InstallLogUtils.getInstallLogger().log(Level.FINEST, "Set log dir to " + (Utils.getLogDir() != null ? Utils.getLogDir() : Utils.getOutputDir()));
+                workareaDirStr = BootstrapConstants.LOC_AREA_NAME_WORKING_UTILS;
             }
-            sb.setUserDir(serverXML.getParentFile().getParentFile().getParentFile());
```

Comment:
I don't know what this call was for on the Builder, but I don't see an equivalent on the new code.

Reply:
That's a mistake.  The usr dir of the EmbeddedServerImpl should be <wlp>/usr.